LICENSE

```diff
@@ -0,0 +1,22 @@
+
+MIT License
+
+Copyright (c) 2017 Sébastien Duperron @liquidseb
```

Comment:
`@liquidseb` is not relevant here. It should be removed or replaced by my GitHub handle `@liquidsoul`